src/encauth/gcm/gcm_memory.c

```diff
@@ -106,6 +106,7 @@ int gcm_memory(      int           cipher,
        err = CRYPT_INVALID_ARG;
     }
 LTC_ERR:
+    err = gcm_reset(gcm);
```

Comment:
```suggestion
    gcm_reset(gcm);
```

You're overwriting `err` here with `CRYPT_OK`, which breaks some expectations and lets CI fail.

Reply:
Thank you very much for your suggestion. I've pushed the update.